.env

```diff
@@ -1,3 +1,3 @@
-KUBE_VERSION=1.30.3
-HELM_VERSION=3.15.4
+KUBE_VERSION=1.31.1
+HELM_VERSION=3.16.0
 YQ_VERSION=4.44.3

```

Comment:
Any updates available for YQ ?

Reply:
Nope: https://github.com/mikefarah/yq/releases